Tileset Sanitizing / Added worlds

Coderwelsch opened the pull request:
- sanitized tilesets from garv
- added haus_2 (made by eddy) / sanitized tilesets
- worked on freiland map

Coderwelsch committed:
Added new sorted tilesets

Coderwelsch committed:
Replaced haus_2 with newer version

Coderwelsch committed:
Updated overview map

Coderwelsch committed:
Removed old datasets

Coderwelsch committed:
Removed old tilesets

Coderwelsch committed:
Removed redundant tsx file

Coderwelsch committed:
Removed radialsystem maps

Coderwelsch committed:
Removed old spartacus version files

Coderwelsch committed:
Renamed latest spartacus map to final name

Coderwelsch committed:
Removed bingo sign

Coderwelsch committed:
Removed boats

Coderwelsch committed:
Removed cccplogo.tsx

Coderwelsch committed:
Removed freiland_massstab_V01.json

Coderwelsch committed:
Added collision layer & changed order of layers

Coderwelsch committed:
Removed temp file

Coderwelsch committed:
Added collisions

Coderwelsch committed:
Added tiles from garv

Coderwelsch committed:
Added tileset_freiland_furnitures

Coderwelsch committed:
Added leaves to tileset_trees.png

Coderwelsch committed:
Renamed spartacus map to haus_3

Coderwelsch committed:
Renamed haus4 to haus_4

Coderwelsch committed:
Renamed main.json to haus_5.json

Coderwelsch committed:
Added animations to tileset_freiland_buildings

Coderwelsch committed:
Added more furnitures

Coderwelsch committed:
Worked on ground plan / freiland area

Coderwelsch committed:
Worked on ground plan / freiland area

Coderwelsch committed:
Removed old tilesets

Coderwelsch committed:
Renamed machbar top floor to haus_5_topfloor

Coderwelsch committed:
Moved animations to extra tileset

Coderwelsch committed:
Removed unused map

Coderwelsch committed:
Updated collision tiles

Coderwelsch committed:
Fixed collision and start action tiles

Coderwelsch committed:
Test

Coderwelsch committed:
Test «Rollback»

Coderwelsch committed:
Added building fronts of haus 2, 3 and 5

Coderwelsch committed:
Worked on overview map

Coderwelsch committed:
Removed «apiUrl» properties from maps

Coderwelsch committed:
Added concrete floor

Coderwelsch committed:
Re-arranged tiles, added some more buildings

Coderwelsch committed:
Added more furnitures

Coderwelsch committed:
Worked on tilemap

Coderwelsch committed:
Worked on freiland map

Coderwelsch committed:
Fixed some overlayed tiles bugs

Coderwelsch committed:
Added more details

Coderwelsch committed:
Added arrows and way marker

edward-loves-github committed:
New Indoor Tileset

Coderwelsch committed:
Removed redundand kra files

Coderwelsch committed:
Added haus_4 front wall

Coderwelsch committed:
Added way out to haus_2

Coderwelsch committed:
Added haus_4 and more trees and deco stuff

Coderwelsch committed:
Removed __DEV layer, moved layer to bottom

Coderwelsch committed:
Removed screenshot

Coderwelsch committed:
Removed colliding walls

Coderwelsch committed:
Worked on freiland map

Coderwelsch committed:
Worked on haus_2

chsterz commented:
I am hesitant to merge this, since (I think) parts of the Map are not working (Sparti / Haus5)

Coderwelsch committed:
Remove unused map »t«

Coderwelsch committed:
Added legacy maps and tilesets

Coderwelsch committed:
Remove legacy maps / Reworked freiland map exitUrls

Coderwelsch committed:
Ups, don’t wanted to remove haus_2 map …

Coderwelsch committed:
Add modified version of haus_1 (added collisions and stuff)

Coderwelsch committed:
Add start layer

Coderwelsch committed:
Add jumper markers

Coderwelsch committed:
Add jumper markers for legacy maps

Coderwelsch committed:
Add additional exit to legacy haus_3

Coderwelsch committed:
Added more collisions

Coderwelsch committed:
Added jumper markers to secret rooms

Coderwelsch committed:
Reworked exit/entry points of maps

Coderwelsch committed:
Add startLayer props to haus_2

Coderwelsch committed:
Remove depth property from haus_5 layer

Coderwelsch committed:
Removed fallingsnow

Coderwelsch committed:
Removed depth prop from haus_3

Coderwelsch committed:
Fix haus_2 jump point

Coderwelsch committed:
Removed depth property

Coderwelsch committed:
Hopefully fixed haus_5 (double ayer name)

Coderwelsch committed:
Fixed haus_2

Coderwelsch committed:
Added cat animation frames

edward-loves-github committed:
Add files via upload

edward-loves-github committed:
Add files via upload

edward-loves-github committed:
Add files via upload

edward-loves-github committed:
Add files via upload

Änderung von Martin

edward-loves-github committed:
Add files via upload

Tedros (Teddy) erklärt wieso Gemüse so aussieht wie es aussieht. Soll als random gag im Haus 2 beim Teddybär eingebaut werden.

edward-loves-github committed:
Add files via upload

edward-loves-github committed:
Add files via upload

Coderwelsch committed:
Added overview map

Coderwelsch committed:
Fixed haus_2 tilesets

Coderwelsch committed:
Added another small grass island

Coderwelsch committed:
Removed big fat text tiles

profemo committed:
fixed minor collisions

Mic droped
walls hardened
Exit secret room fixed

Coderwelsch committed:
Fixed issues on haus_4 (collision, transparent layers)

profemo committed:
minor changes

changed entry sparti
added decoration

profemo committed:
killed gruendungscampus

more hamacas
bees added
gruendercampus destroeyd

Coderwelsch committed:
Fixed haus_1 issues

Coderwelsch committed:
Added jump markers for second floor of haus_1, Added haus_1_obergeschoss with collisions and stuff

profemo committed:
sparti cleaned

Coderwelsch committed:
Improved room transition-walk ways

profemo committed:
feature extase

Bienen!
Fische!
mit OpenWebsite
dazu Eingang von sueden bei Haus2

profemo committed:
minor bugfixes

Eingang Haus2 repariert
Eingang haus 4 reduziert
Mehr blumen
Bauminsel mit g...weg

profemo committed:
exits changed

rakete geloescht
bissel deko weg
Map kleiner
neue Eingaenge
neue Ausgaenge

profemo committed:
minor changes

rakete geloescht
bissel deko weg
Map kleiner
neue Eingaenge
neue Ausgaenge

Coderwelsch committed:
Add tileset_freiland_signs

Coderwelsch committed:
Add tileset_freiland_signs

Coderwelsch committed:
Delete haus_3.json

Coderwelsch committed:
Add haus_3 update

Coderwelsch committed:
Add new tilesets to legacy

chsterz committed:
Haus 4 Exit

Coderwelsch committed:
Add info-i animation frames to tileset_animations

Coderwelsch committed:
Added mahnmal to tileset_freiland_buildings

profemo committed:
Deko Added

transparenz runter von Exits
deko und blumen verteilt

Coderwelsch committed:
Add bicycle-ring sound and new sounds dir

Coderwelsch committed:
Moved teddy sound file to sounds dir

Coderwelsch committed:
Add teddy-corona sound file

chsterz committed:
LiveStream (Christophs) im Sparti

Coderwelsch committed:
Added more teddy sound files

profemo committed:
radio exits

frapo drin
neuer geheimgang
collision ergaenzt
deko

chsterz committed:
Add start.html

profemo committed:
fixed soundz

spuelung installiert
radio gespielt

chsterz committed:
Change Sparti Dancefloor URL to Freiland Channel

profemo committed:
sound fixed

Coderwelsch committed:
Add freiland postcard

Coderwelsch committed:
Added 2d freiland postcard to tileset_freiland_furnitures

Coderwelsch committed:
Added postcard to haus_3

Coderwelsch committed:
Added info icon

Coderwelsch committed:
Added mahnmal website

Coderwelsch committed:
Improved mahnmal

Coderwelsch committed:
Added footprints to freiland

chsterz committed:
Back to Audio-Stream on Sparti Dänse

chsterz committed:
Remove Startpage in Landingzone

chsterz committed:
Add Freiland Channel Stream to Haus2

Coderwelsch committed:
Added more signs to tileset_freiland_signs

Coderwelsch committed:
Added disco ball thingii and flags to tileset_animations

edward-loves-github committed:
Add files via upload

Dokumentation:
tileset_indoor wurden Treppen hinzugefügt
haus_1 wurden Treppen, start und exit angepasst
haus_1_obergeschoss wurden Treppen, start und exit angepasst

edward-loves-github committed:
Add files via upload

Coderwelsch committed:
Added tiles

Coderwelsch committed:
Moved sound files to sounds dir

Coderwelsch committed:
Removed old sounds

Coderwelsch committed:
Removed tileset_spartacus

profemo committed:
minor changes

- mini jitsi Raeume geloescht
- sounds hinzugefuegt

profemo committed:
Signs added

cafe added
solikante added
frapo sign added

chsterz committed:
Sparti nun Stream

chsterz committed:
haus2 pingus again

chsterz committed:
CultusOffice

chsterz committed:
Make office startLayer

chsterz committed:
Fix H1 Main exit

chsterz committed:
Fix H1 Main Treppe

chsterz committed:
Fix Main Downstairs H1

chsterz committed:
Fix H1 Atelier Exit

chsterz committed:
Fix H1 Atelier Exit

Coderwelsch committed:
Added pincode to seminarraum / open & close doors, improved collisions

chsterz committed:
Haus1 OG Flur Stream

chsterz committed:
Sparti Playlist on Sparti Dancefloor

Coderwelsch committed:
Added signs to haus_1

Coderwelsch committed:
Added signs to haus_1_obergeschoss

Coderwelsch committed:
Added pincode for seminar room #2

chsterz committed:
Haus1 Flur Oben Videoaufzeichnung statt Stream

chsterz committed:
Haus1 Flur Oben Teppich Jüdische Architekten

profemo committed:
Treppen fix

Treppe Offenes Atelier
mehr Schilder
Seminarraeume mit jitsi ausruesten

Coderwelsch committed:
Added tileset_animation to all maps

Coderwelsch committed:
Removed redundant haus_4 from legacy

Coderwelsch committed:
Added new haus_3 map (bare)

Coderwelsch committed:
Fixed pincodes and jitsi room in haus_1_obergeschoss

Coderwelsch committed:
Added fun with flags

chsterz committed:
Haus1 Oben: Fix Treppe links

chsterz committed:
Haus1 Oben: Fix Treppe links

Coderwelsch committed:
Added cute little tiny citties for overall satisffaction <3

Coderwelsch committed:
Added homepages of artists

Coderwelsch committed:
Added homepage links to rooms

chsterz committed:
Graffitti Test

chsterz committed:
Grafittitest 2

chsterz committed:
77 Graffiti-Random Tileset

chsterz committed:
Fix Graffitirandom

chsterz committed:
Fix?

chsterz committed:
Fix Graffiti

chsterz committed:
Graffitis breiter

chsterz committed:
Graffitti: Made Grafitti continous

chsterz committed:
Quickfix Graffiti for current pic

chsterz committed:
Sound outside Sparti

chsterz committed:
Graffiti shuffle

chsterz committed:
Really make shuffle work

chsterz committed:
Deleted intermediate file

chsterz committed:
Tarzan, System Schulz, Noch kein Schild

Coderwelsch committed:
Added tree behind graffiti wall

chsterz committed:
Sound Game Synthie

chsterz committed:
Add Synthie URL

chsterz committed:
info i

chsterz committed:
Skribbl

Coderwelsch committed:
Added start_Youtube_Livestream_1 layer to freiland.json

profemo committed:
add new features

new games
new cats
smaller map
new exit
new deko

profemo committed:
new features

new games
new cats
smaller map
new exit
new deko

Coderwelsch committed:
Added big livestream button

Coderwelsch committed:
Fixed map start link (ungrouped layer)

Coderwelsch committed:
Renamed yt jump point / start

Coderwelsch committed:
Separated jump layer / yt livestream url island

Coderwelsch committed:
Added startLayer property

Coderwelsch committed:
Removed gruendungscampus talk area

Coderwelsch committed:
Possibly fixed youtube embed url for yt livestream island

Coderwelsch committed:
Possibly fixed youtube embed url for yt livestream island

profemo committed:
strombad wieder angelegt

chsterz committed:
Silvesterstream Sparti

chsterz committed:
Silvesterstream Sparti correct channel ID

chsterz committed:
Silvesterstream Sparti correct channel ID, embed URL

chsterz committed:
Silvesterstream Haus2, embedded URL

chsterz committed:
Silvesterstream Haus2, teste mal anders

chsterz committed:
Silvesterstream Haus2, teste mal anders

chsterz committed:
Sparti URL noch besser, mit Vorschau

chsterz committed:
Freilandplenum Footsteps

chsterz committed:
properly name jitsiroom

chsterz committed:
remove machbarbank

chsterz committed:
mark jitsis H5

chsterz committed:
rename machbarbank back to machbarbank, dont support Raucherecken

chsterz committed:
try to fix door

chsterz committed:
try to fix door2

chsterz committed:
try to fix door3

chsterz committed:
try to fix door4

chsterz committed:
try to fix door5

chsterz committed:
Einrichtung Semi 1 Anfrage

chsterz committed:
Try to fix pincode

chsterz committed:
Try to fix pincode 2

chsterz committed:
Try to fix pincode backwards

chsterz committed:
Semi 2 Tür AutoOpen, No Code

chsterz committed:
Remove misleading Semi2 Door

chsterz committed:
Video only on 2nd Half

chsterz committed:
Remove 1 more Architecture Carpet

chsterz committed:
Remove Architecture Video Completely

Coderwelsch committed:
Added stands and char animation

chsterz committed:
Teppiche für die Veranstaltungen hinzugefügt

Coderwelsch committed:
Moved stands / added deco and stand markers
